waltz-server/src/main/java/com/wepay/waltz/server/internal/WaltzServerHandler.java

```diff
@@ -235,8 +235,15 @@ public void onChannelActive() {
         @Override
         public void onChannelInactive() {
             synchronized (partitions) {
+                List<Integer> partitionsNotRemoved = new ArrayList<>();
```

Comment:
Sorry for nit picking..
Better to use `partitionClientsNotRemoved` instead of `partitionsNotRemoved`.